website/src/pages/home/_Hero.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FunctionComponent, useEffect, useState } from 'react'
+import { Link } from 'gatsby'
+import classNames from 'classnames'
+
+import { CustomerLogos } from '../../components/CustomerLogos'
+import { buttonStyle, buttonLocation } from '../../tracking'
+
+import meshLeft from './assets/mesh-left.png'
+import meshRight from './assets/mesh-right.png'
+
+import styles from './home.module.scss'
+
+const Hero: FunctionComponent = () => {
+    const headlines: string[] = ['Understand and search', 'Fix vulnerabilities and issues', 'Automate key workflows']
+    const [headlineIndex, setHeadlineIndex] = useState(0)
+    const [headline, setHeadline] = useState(headlines[0])
+
+    useEffect(() => {
+        const cycle = setInterval(() => {
+            const newIndex = headlineIndex === headlines.length - 1 ? 0 : headlineIndex + 1
+
+            setHeadline(headlines[newIndex])
+            setHeadlineIndex(newIndex)
+        }, 5000)
+
+        return () => clearInterval(cycle)
+    })
+
+    return (
+        <div className="bg-white py-7 px-2 position-relative">
+            <div
+                className={classNames(styles.mesh, 'd-none d-lg-block position-absolute top-0 left-0')}
+                style={{ backgroundImage: `url(${meshLeft})` }}
+            />
+            <div
+                className={classNames(styles.mesh, 'd-none d-lg-block position-absolute top-0 right-0')}
+                style={{ backgroundImage: `url(${meshRight})` }}
+            />
```

Comment:
These look really sharp! I noticed shadows when testing in Safari that aren't present in Chrome or Firefox. It looks like the linear-gradient added on `mesh ::after` displays black as the transparent value. I found some discussion of this [here](https://stackoverflow.com/a/56548711) that suggests this is a difference specific to Safari. The effect looks consistent across browsers if `rgba(0, 0, 0, 0)` is swapped for `rgba(255, 255, 255, 0)`.

<img width="600" alt="Screen Shot 2022-04-21 at 1 28 36 PM" src="https://user-images.githubusercontent.com/60713139/164529148-e2415398-fc05-4943-95c6-59f5a8240da4.png">

Reply:
Wow, nice catch @zlonko! I didn't test this in Safari until now but this is a good reminder that we should add this to our PR descriptions to test Chrome, Safari, and Firefox going forward. We also have access to Browserstack. ✨ 

We should also add a githook for PR description templates and add this under our `### Test` section as a reminder.

cc: @st0nebraker